test/controllers/images_controller_test.rb

```
@@ -9,4 +9,20 @@ class ImagesControllerTest < ActionDispatch::IntegrationTest
       assert_select 'input[type=submit]'
     end
   end
+
```

What will happen if we are trying to access an image with invalid id, for example `id=-1`. Can we assert some error will raise in an invalid test case?